src/token/token_service_repo.ml

```diff
@@ -50,13 +50,34 @@ struct
           $6,
           $7
         )
-|sql}
+        |sql}
 
     let insert connection ~token =
       let module Connection = (val connection : Caqti_lwt.CONNECTION) in
       Connection.exec insert_request token
       |> Lwt_result.map_err Caqti_error.show
 
+    let update_request =
+      Caqti_request.exec Model.t
+        (* TODO Check if correct sql syntax*)
+        {sql|
+        UPDATE token_tokens
+        SET
+          uuid = $1,
+          token_value = $2,
+          token_data = $3,
+          token_kind = $4,
+          status = $5,
+          expires_at = $6,
+          created_at = $7
+        WHERE token_tokens.token_value = ?
```

Comment:
@aronerben Here you should use `$2` to reference the token_value

Reply:
Done